src/lib.rs

```diff
@@ -131,7 +152,7 @@ impl<Fs: FileSystem> ResolverGeneric<Fs> {
     ///
     /// A specifier is the string passed to require or import, i.e. `require("specifier")` or `import "specifier"`.
     ///
-    /// The path must be an **absolute** path where the specifier is resolved from.
+    /// `path` must be an **absolute** path to a directory where the specifier is resolved against.
```

Comment:
nit: we could - in a different PR - add a `debug_assert!` that checks if the path is absolute